api/server/sdk/cluster_pair.go

```diff
@@ -0,0 +1,134 @@
+/*
+Package sdk is the gRPC implementation of the SDK gRPC server
+Copyright 2018 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package sdk
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/libopenstorage/openstorage/api"
+	"github.com/libopenstorage/openstorage/cluster"
+)
+
+// ClusterPairServer is an implementation of the gRPC OpenStorageClusterServer interface
+type ClusterPairServer struct {
+	cluster cluster.Cluster
+}
+
+// Create a new cluster with remote pair
+func (s *ClusterPairServer) Create(
+	ctx context.Context,
+	req *api.ClusterPairCreateRequest,
+) (*api.ClusterPairCreateResponse, error) {
+
+	if len(req.GetRemoteClusterIp()) == 0 {
```

Comment:
These checks aren't required here. This should just be a pass through. The implementation will do the checks. Same for APIs below